sdks/java/core/src/main/java/org/apache/beam/sdk/io/AvroIO.java

```diff
@@ -327,6 +328,14 @@
       }
 
       @Override
+      public void populateDisplayData(DisplayData.Builder builder) {
+        super.populateDisplayData(builder);
+        builder
+          .addIfNotNull("filePattern", filepattern)
```

Comment:
oh never mind I'm forgetting our discussion. And no, it's not, it's in the FileBasedSource inside.